### PR TITLE
[CCTRI-2808] Implement observe/observables endpoint & sightings mapping

### PR DESCRIPTION
**Pull request by mstoro**

### CODE REVIEW

File: code/api/enrich.py [hunk 1]
```diff
@@ -17,9 +27,24 @@ def deliberate_observables():
...
-    return jsonify_data({})
+    credentials = get_credentials()
+    observables = filter_observables(get_observables())
+
+    g.sightings = []
+    client = GraylogClient(credentials)
+
+    for observable in observables:
+        result = client.get_data(observable)
+        messages = (result['results'][client.query_id]['search_types']
```
Comment (oshynk) [inline code/api/enrich.py:38]: Why can't you do this inside the client?

Comment (oshynk) [reply code/api/enrich.py:38]: I just have a concern about query_id and search_type_id. If you call get_data() twice you will lose this info for the first call. From my perspective, you need either move this part to the client and then no need in these attributes or make get_data() to return it, so every new call will return the correct id

Comment (mstoro) [reply code/api/enrich.py:38]: I need new query_id and search_type_id for each observable. I'll move this part into the client, to get rid of calling attributes.

Comment (mstoro) [reply code/api/enrich.py]: https://github.com/CiscoSecurity/tr-05-serverless-graylog/pull/4/commits/21fca4c40bb7c11b52b25fbc46a7be1dd640033e

File: code/tests/unit/api/test_enrich.py [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
 from http import HTTPStatus
-from .utils import get_headers
 from unittest.mock import patch
+
+from pytest import fixture
+
+from .utils import get_headers
 from ..conftest import mock_api_response
-from ..payloads_for_tests import EXPECTED_RESPONSE_OF_JWKS_ENDPOINT
+from ..payloads_for_tests import (
```
Comment (oshynk) [inline code/tests/unit/api/test_enrich.py:8]: One more thing. We agreed to use absolute imports

Comment (mstoro) [reply code/tests/unit/api/test_enrich.py]: https://github.com/CiscoSecurity/tr-05-serverless-graylog/pull/4/commits/21daad2b0363d671703827f81a24a1c412c99b1a

File: code/tests/unit/api/test_enrich.py [hunk 3]
```diff
@@ -41,14 +46,19 @@ def test_enrich_call_with_valid_jwt_but_invalid_json_value(
 
 @fixture(scope='module')
 def valid_json():
-    return [{'type': 'domain', 'value': 'cisco.com'}]
+    return [{'type': 'domain', 'value': '24.141.154.216'}]
```
Comment (MaxMhonc) [inline code/tests/unit/api/test_enrich.py:49]: I can have lack info, but I have to ask. Does it correct type - "domain" and value - "ip"?

Comment (mstoro) [reply code/tests/unit/api/test_enrich.py]: That's not correct, thanks!
Fixed it https://github.com/CiscoSecurity/tr-05-serverless-graylog/pull/4/commits/17f0f08278565774f43c1e5b2e507400d8f99ab8

**Review by oshynk — APPROVED** [review]

**Review by MaxMhonc — APPROVED** [review]

**Pull request merged**